File: portal-api/portal-api/Model/PictureEntry.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace portal_api.Model
{
    public class PictureEntry
    { 
        [Key]
        [ScaffoldColumn(false)]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int PictureId { set; get; }

        [Required]
        public string Title { set; get; }
        
        public string Tag { set; get; }

        [Required]
        public DateTime CreationDate { set; get; }

        public int? CreatorUserId { set; get; }
        [Required]
        public PortalUser Creator { set; get; }

        [Required]
        public byte[] Image { set; get; }
    }
}

File: picture-portal/src/app/api/models/picture-entry.ts
/* tslint:disable */
/* eslint-disable */
import { PortalUser } from './portal-user';
export interface PictureEntry {
  creationDate: Date;
  creator?: PortalUser;
  image: string;
  pictureId?: number;
  tag?: null | string;
  title: string;
}
